littlebot_control/launch/spawn_diffdriver_controller.launch.py

```diff
@@ -0,0 +1,49 @@
+import os
+from launch_ros.actions import Node
+from launch import LaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.actions import RegisterEventHandler
+from launch.event_handlers import OnProcessExit
+
+
+def generate_launch_description():
+    joint_state_broadcaster_spawner = Node(
+        package="controller_manager",
+        executable="spawner",
+        arguments=["joint_state_broadcaster", "--controller-manager", "/controller_manager"],
```

Comment:
line too long (93 > 79 characters)
```suggestion
        arguments=["joint_state_broadcaster",
                               "--controller-manager",
                               "/controller_manager"],
```

Reply:
fixed [here](https://github.com/NestorDP/littlebot/pull/10/commits/33db77a6b46130d550d8f299364a59b44734b9e4)